src/components/Input/Input.tsx

```diff
@@ -67,11 +89,22 @@ class Input extends UIComponent<any, any> {
     type: 'text',
   }
 
+  static autoControlledProps = ['value']
+
   inputRef: any
 
-  computeTabIndex = props => {
-    if (!_.isNil(props.tabIndex)) return props.tabIndex
-    if (props.onClick) return 0
+  state: any = { value: this.props.value || '' }
+
+  getInitialAutoControlledState() {
+    return { value: '' }
+  }
```

Comment:
The `state = {}` on line 96 seems to conflict with the `getInitialAutoControlledState` here on 98.  The result of `getInitialAutoControlledState` is applied after the state member.